### DISCUSSION TITLE
[Project Beta] Cannot select sorting order of "Group by"

### DISCUSSION THREAD

**Discussion by cschreib** [community/community#13842] [Projects and Issues]
I want to sort the content of a project first by status (todo at top, done at bottom), then priority (highest priority at top), then by size (largest at top). I cannot do this with the current UI. However I can:
 - hide closed items (effectively does "sort by status", as I only use "Todo" and "Done")
 - group by priority
 - sort by decreasing size

That's almost good, except "Group by" does not allow me to choose the order of the groups. Consequently, I currently have the lowest priority items displayed at the top, and no way to change this, short of re-doing all the priority labeling to invert the order of the priorities in the field...

This would be less of an issue if multi-column sorting was possible, but I still think the group order should be configurable in some way.

**Comment by droumis** [comment]
This seems like an impactful improvement that would be very easy to implement

**Comment by kohle** [comment]
This would be a great improvement.

**Comment by leape** [comment]
Yes please! +1

**Comment by Munhuggas** [comment]
If me commenting here makes this more prioritized we would also be very happy if we could change the sort order of the group swim lanes.

**Comment by mvdan** [comment]
I figured out I can edit the order of the field in the settings to do this. For example, I had priorities as P2, P1, P0, but I wanted "group by" to show P0 first. Reordering in the settings fixed the views.
